stdlib/jdbc/src/main/java/org/ballerinax/jdbc/actions/ExternActionMethods.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * you may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinax.jdbc.actions;
```

Comment:
Since we have only one class now, shall we move both `ExternActionMethods` and `ExternFunctionMethods` to a single java package like `org.ballerinax.jdbc.methods` and rename the above classes: `ExternActionMethods` -> `ExternActions` and `ExternFunctionMethods` -> `ExternFunctions`?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/20109/commits/1f71769c3d1f73358a70b654c7387065fac1807c